2013-01-14 23:19:39.608000: ENGINE starting
2013-01-14 23:19:39.609000: ENGINE loading STUDENT ENGINE part 2
2013-01-14 23:19:39.619000: ENGINE loaded player module: OKComputer as player 1
2013-01-14 23:19:39.619000: QUORIDOR ENGINE CLIENT 1.0.5 by Adam Oest initializing
2013-01-14 23:19:40.110000: ENGINE calling STUDENT ENGINE init
2013-01-14 23:19:40.111000: STUDENT ENGINE init call successfully returned
2013-01-14 23:19:40.111000: ENGINE calling init() for player 1
2013-01-14 23:19:40.112000: OKComputer initialized as player 1
2013-01-14 23:19:40.134000: ENGINE GUI accessible at:
https://adelaide.cs.rit.edu/~royale/quoridorui/gameview/gameid.40523/partone.0/apikey.36bc8c65d1d4af66485/public.ee0af6267ccf87f41878a5fabcc69df02f1b642c50f4d8d9a1b29/animations.500/
2013-01-14 23:19:40.134000: ENGINE processing move # 1
2013-01-14 23:19:40.134000: ENGINE getting next move from player 1
2013-01-14 23:19:40.137000: ENGINE validating player 1 move. Player 1 Move: [8,4] -> [7,4]
2013-01-14 23:19:40.137000: ENGINE calling STUDENT ENGINE validate_move
2013-01-14 23:19:40.137000: ERROR: 8
2013-01-14 23:19:40.138000: TRACE: 
Traceback (most recent call last):
  File "C:\Users\Adam\Desktop\a\Quoridor_Client_2121\Releases\1.0.5\Python3.2\Model\__init__.py", line 560, in studentEngineCall
  File "C:\Users\Joe\Documents\GitHub\Quoridor\StudentEngines\jak3122\__init__.py", line 246, in validate_move
    if not engineData.board.squares[r][c] == player_id:
KeyError: 8

2013-01-14 23:19:40.138000: ERROR: 2013-01-14 23:19:40.138000: ENGINE exiting due to STUDENT ENGINE exception

2013-01-14 23:19:40.139000: TRACE: 
Traceback (most recent call last):
  File "C:\Users\Adam\Desktop\a\Quoridor_Client_2121\Releases\1.0.5\Python3.2\Engine\controller.py", line 246, in nextTurn
  File "C:\Users\Adam\Desktop\a\Quoridor_Client_2121\Releases\1.0.5\Python3.2\Model\__init__.py", line 567, in studentEngineCall
Engine.security.GameException: 2013-01-14 23:19:40.138000: ENGINE exiting due to STUDENT ENGINE exception


2013-01-14 23:19:40.370000: 2013-01-14 23:19:40.138000: ENGINE exiting due to STUDENT ENGINE exception

2013-01-14 23:19:40.370000: ENGINE finished with next move
2013-01-14 23:19:40.370000: ENGINE shutting down
